pkg/common/kafka/sarama/sarama_test.go

```diff
@@ -76,6 +76,62 @@ dispatcher:
   retryInitialIntervalMillis: ` + commontesting.DispatcherRetryInitial + `
   retryTimeMillis: ` + commontesting.DispatcherRetry + `
 `
+
+	EKDefaultSaramaConfigWithRootCert = `
+Net:
+  TLS:
+    Enable: true
+    Config:
+      RootPEMs: # Array of Root Certificate PEM Files As Strings (Mind indentation and use '|-' Syntax To Avoid Terminating \n)
+      - |-
+        -----BEGIN CERTIFICATE-----
+        MIIGBDCCA+ygAwIBAgIJAKi1aEV58cQ1MA0GCSqGSIb3DQEBCwUAMIGOMQswCQYD
```

Comment:
Hope this isn't a real cert

Reply:
It is a real cert - but it is just the SAP CloudPlatform RootCert that identifies them and is not a security concern.  Since it will be loaded/parsed into the CertPool struct I need a valid one.